src/js/providers/flash.js

```diff
@@ -222,7 +222,6 @@ define([
                         this.sendEvent(events.JWPLAYER_MEDIA_ERROR, {
                             message: 'Error loading media: File could not be played'
                         });
-                        this.setState(states.IDLE);
```

Comment:
Will that create a new event / player state in the public API? we should avoid that.